c/include/libsbp/v4/signing/MSG_ECDSA_SIGNATURE.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2015-2021 Swift Navigation Inc.
+ * Contact: https://support.swiftnav.com
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+/*****************************************************************************
+ * Automatically generated from yaml/swiftnav/sbp/signing.yaml
+ * with generate.py. Please do not hand edit!
+ *****************************************************************************/
+
+#ifndef LIBSBP_V4_SIGNING_MSG_ECDSA_SIGNATURE_H
+#define LIBSBP_V4_SIGNING_MSG_ECDSA_SIGNATURE_H
+
+#include <math.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <libsbp/common.h>
+#include <libsbp/signing_macros.h>
+#include <libsbp/v4/string/sbp_string.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/******************************************************************************
+ *
+ * SBP_MSG_ECDSA_SIGNATURE
+ *
+ *****************************************************************************/
+/** An ECDSA signature
+ *
+ * An ECDSA-256 signature using SHA-256 as the message digest algorithm.
+ */
+typedef struct {
+  u8 flags;
```

Comment:
Should we document here how the flags byte it's going to be used?

Reply:
Fixed.